### ISSUE TITLE
Python plugins don't works in Autopsy for Linux

### ISSUE THREAD

**Issue by nannib** [sleuthkit/autopsy#4170]
I put the directories extracted from zip files of external Autopsy plugins, e.g. https://github.com/markmckinnon/Autopsy-Plugins
in /home/caine/.autopsy/dev/python_modules
But nothing to do!
They never appear in the "Run Ingest modules", in Windows they works but in Linux (Ubuntu 18.04) nothing.
Am I wrong?

**Comment by markmckinnon** [comment]
Is the directory /home/caine/.autopsy/dev/python_modules what shows up if you are in Autopsy and go to the menu tools -> python plugins?  I know not all the plugins will work in Autopsy and I have started the process to make them compatible with Linux.

**Comment by nannib** [comment]
Yes, it is. All external python plugins do not appear,  not only yours ;-)
I used yours just for instance in this thread.

**Comment by markmckinnon** [comment]
I will take a look and see if I can recreate the issue.  Autopsy 4.6 for Linux I presume and Ububtu 18.04 from what you said above.

**Comment by nannib** [comment]
Autopsy 4.8 and Ubuntu 18.04

**Comment by nannib** [comment]
I noticed another strange thing, if I run Autopsy 4.8 in Ubuntu 18.04 as normal user and I click Python Plugins, I get the browser directory window.
If I run as root (sudo autopsy) and I click Python Plugins, I get Gwenview for browsing directories, I don't know why root user open it! I checked mimetypes.list but nothing, only with Autopsy I get this...

**Comment by bcarrier** [comment]
@nannib can you look in the log file to see if there are any exceptions / errors?    I'm honestly not sure we tested this, but I think we do some file / path manipulation and I wonder if there are some WIndows assumptions in there.

**Comment by nannib** [comment]
[messages.log](https://github.com/sleuthkit/autopsy/files/2434238/messages.log)
[autopsy.log.0.txt](https://github.com/sleuthkit/autopsy/files/2434239/autopsy.log.0.txt)

[solr.log.stderr.txt](https://github.com/sleuthkit/autopsy/files/2434240/solr.log.stderr.txt)

**Comment by dkarpo** [comment]
Hello.  After Nanni and I resolved issue #4166 I tested the Python plugins as a regular 'caine' user (/home/caine/.autopsy/dev/plugins) and as root (/root/.autopsy/dev/plugins) and in both cases they are being picked up by Autopsy 4.8.0 correctly.  Nanni, can we close this issue too?
